src/message/__tests__/messageActionSheet-test.js

```diff
@@ -56,60 +56,98 @@ describe('constructActionButtons', () => {
   });
 });
 
-describe('constructHeaderActionButtons', () => {
+describe('constructTopicActionButtons', () => {
+  test('show mark as read if topic is unread', () => {
+    const stream = eg.makeStream({ name: 'test stream', stream_id: 123 });
+    const subscriptions = [{ ...eg.subscription, ...stream }];
+    const unreadStreams = eg.makeUnreadStreamsState([
+      { streamId: 123, topics: [{ topicName: 'test topic', messageIds: [1, 2] }] },
```

Comment:
Instead of saying `streamId: 123` here and `stream_id: 123` above where `stream` is constructed, this can say `streamId: stream.stream_id` here. That way the connection between this and the ID of `stream` is explicit.

I think that also makes it unnecessary to add a `stream_id` option to `eg.makeStream`. In general I think in almost any place we might use that, it'd be better to turn the flow around just like here: let the stream get whatever ID it gets, and where we want to have that stream's ID just refer to it directly.